featuretools/tests/primitive_tests/test_features_serializer.py

```diff
@@ -45,14 +45,14 @@ def test_base_features_in_list(es):
 
 
 def test_multi_output_features(es):
-    value = ft.IdentityFeature(es['log']['product_id'])
+    value = ft.IdentityFeature(es, 'log', 'product_id')
     threecommon = ft.primitives.NMostCommon()
-    tc = ft.Feature(es['log']['product_id'], parent_entity=es["sessions"], primitive=threecommon)
+    tc = ft.Feature(ft.IdentityFeature(es, 'log', 'product_id'), parent_dataframe_name="sessions", primitive=threecommon)
```

Comment:
Use `value` in place of `ft.IdentityFeature(es, 'log', 'product_id')` here? And should value be named something else?